custom/icds_reports/tests/outputs/agg_awc-st1-2017-05-01.csv

```diff
@@ -1,27 +1,27 @@
-st1,d1,b2,s3,a27,2017-05-01,1,7,,,,,,,,,,,,,,,,,1,,12,12,,,,,,,,,,,1,2,16,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,2017-05-06,,,,,,1,0,1,,1,1,1,1,,0,,1,5,19,,,,0,,,,,no,,,,,,5,,,,,,134,8,8,,1,2,16,0,0,0,0,1,st1,d1,b2,a27,a27
-st1,d1,b2,s4,a28,2017-05-01,1,5,,,,,,,,,,,,,,,,,1,,0,17,,,,,,,,,,,1,3,22,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,0,,0,0,26,,,,0,,,,,yes,,,,,,5,,,,,1,87,7,7,,1,3,22,0,0,0,0,,st1,d1,b2,a28,a28
-st1,d1,b2,s3,a35,2017-05-01,1,12,,,,,,,,,,,,,,,,,1,,5,5,,,,,,,,,,,1,1,9,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,2017-05-29,semi_pucca,,,,,1,0,0,,0,0,0,1,,0,,1,0,11,,,,0,,,,,yes,,,,,,5,,,,,1,144,13,13,,1,1,9,0,0,0,0,0,st1,d1,b2,a35,a35
-st1,d1,b2,s4,a36,2017-05-01,1,17,,,,,,,,,,,,,,,,,1,,3,40,,,,,,,,,,,1,2,52,0,0,0,0,0,0,0,0,0,0,0,0,1,0,,,,,,,,,,,,,,,,,,,,,,,,,,,,2017-05-24,,,,,,1,1,0,,0,0,0,1,,0,,1,0,55,,,,0,,0,0,0,yes,,,,,,5,0,0,0,0,1,101,60,60,,1,2,52,3,0,3,0,1,st1,d1,b2,a36,a36
-st1,d1,b2,s3,a43,2017-05-01,1,13,,,,,,,,,,,,,,,,,1,,19,21,,,,,,,,,,,0,1,26,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,0,,0,10,27,,,,0,,,,,yes,,,,,,5,,,,,1,195,31,31,,0,1,26,1,0,1,0,,st1,d1,b2,a43,a43
-st1,d1,b2,s4,a44,2017-05-01,1,10,,,,,,,,,,,,,,,,,1,,8,30,,,,,,,,,,,8,10,30,0,0,36,2,0,0,1,0,1,0,0,0,0,1,,,,,,,,,,,,,,,,,,,,,,,,,,,,2017-05-20,semi_pucca,,,,,1,1,1,,1,1,1,1,,0,,1,1,48,,,,0,,0,0,0,no,,,,,,5,0,0,0,0,0,106,24,24,,8,10,30,2,1,2,1,1,st1,d1,b2,a44,a44
-st1,d1,b1,s1,a1,2017-05-01,1,18,,,,,,,,,,,,,,,,,1,,,,,,,,,,,,,,2,3,,2,0,0,0,0,0,0,0,0,0,0,0,0,0,,,,,,,,,,,,,,,,,,,,,,,,,,,,2017-05-13,semi_pucca,,,,,1,1,0,,0,0,0,1,,0,,1,1,5,,,,0,,0,0,0,no,,,,,,5,0,0,0,0,0,139,1,1,,2,3,,0,0,0,0,0,st1,d1,b1,a1,a1
+st1,d1,b1,s2,a2,2017-05-01,1,10,,,,,,,,,,,,,,,,,1,,10,18,,,,,,,,,,,6,1,30,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,2017-01-19,pucca,,,,,1,0,1,,0,0,1,1,,,,1,2,37,,,,0,,,,,no,,,,,,5,0,0,0,0,0,,,,,6,1,30,,,,,1,st1,d1,b1,a2,a2
+st1,d1,b1,s1,a9,2017-05-01,1,18,,,,,,,,,,,,,,,,,1,,0,4,,,,,,,,,,,3,0,8,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,2017-01-12,,,,,,1,1,1,,0,1,1,1,,,,1,0,11,,,,0,,,,,no,,,,,,5,0,0,0,0,0,,,,,3,0,8,,,,,1,st1,d1,b1,a9,a9
+st1,d1,b1,s2,a10,2017-05-01,1,8,,,,,,,,,,,,,,,,,1,,0,44,,,,,,,,,,,0,2,62,1,0,0,0,0,0,0,0,0,0,0,0,0,0,,,,,,,,,,,,,,,,,,,,,,,,,,,,2017-01-19,pucca,,,,,1,0,1,,0,0,1,1,,0,,1,2,64,,,,0,,0,0,0,yes,,,,,,5,1,1,1,1,1,188,8,8,,0,2,62,0,0,0,0,1,st1,d1,b1,a10,a10
+st1,d1,b1,s1,a17,2017-05-01,1,11,,,,,,,,,,,,,,,,,1,,7,7,,,,,,,,,,,0,3,14,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,0,,0,12,17,,,,0,,,,,no,,,,,,5,0,0,0,0,0,,2,2,,0,3,14,0,0,0,0,,st1,d1,b1,a17,a17
+st1,d1,b1,s2,a18,2017-05-01,1,17,,,,,,,,,,,,,,,,,1,,27,27,,,,,,,,,,,5,6,32,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,0,,0,27,43,,,,0,,,,,yes,,,,,,5,1,1,1,1,1,214,28,28,,5,6,32,0,0,0,0,,st1,d1,b1,a18,a18
```

Comment:
There was some this is changed because there was inconsistency. between num_launched_awcs and num_launched_<location above awc> at aggregation level=5. When  num_launched_awcs is =1 for some awc its num_launched_<location above awc> has to be 1.